constants.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BBOX_GLOBAL = [-180, -90, 180, 90]
+SAS_TOKEN_DEV = os.getenv("DSCI_AZ_SAS_DEV")
+SAS_TOKEN_PROD = os.getenv("DSCI_AZ_SAS_PROD")
+CONTAINER_GLOBAL = "global"
```

Comment:
I think these azure specific settings should be specified elsewhere.

Reply:
Where do you think would be more appropriate?

Reply:
Sorry I missed the comments following this up. I changed this on my branch, it's more along the lines of just having the one SAS_TOKEN and using that instead of the dev/prod split.